### PR TITLE
Add XML and IOD data type

### PR DESCRIPTION
**Pull request by LevZaplatin**

### CODE REVIEW

File: src/Phinx/Db/Adapter/PostgresAdapter.php [hunk 1]
```diff
@@ -1321,7 +1327,16 @@ public function getAllSchemas()
...
-        return array_merge(parent::getColumnTypes(), ['json', 'jsonb', 'cidr', 'inet', 'macaddr', 'interval']);
+        return array_merge(parent::getColumnTypes(), [
+            'json',
+            'jsonb',
+            'xml',
+            'oid',
+            'cidr',
+            'inet',
+            'macaddr',
+            'interval'
```
Comment (garas) [inline src/Phinx/Db/Adapter/PostgresAdapter.php]:
```suggestion
            'interval',
```
Add comma for future diffs to be less noisy

File: src/Phinx/Db/Adapter/MysqlAdapter.php [hunk 2]
```diff
@@ -45,7 +45,13 @@
 class MysqlAdapter extends PdoAdapter implements AdapterInterface
 {
 
-    protected $signedColumnTypes = ['integer' => true, 'biginteger' => true, 'float' => true, 'decimal' => true, 'boolean' => true];
+    protected $signedColumnTypes = [
+        'integer' => true,
+        'biginteger' => true,
+        'float' => true,
+        'decimal' => true,
+        'boolean' => true
```
Comment (garas) [inline src/Phinx/Db/Adapter/MysqlAdapter.php]:
```suggestion
        'boolean' => true,
```
Add comma for future diffs to be less noisy

File: src/Phinx/Db/Adapter/PostgresAdapter.php [hunk 3]
```diff
@@ -1321,7 +1327,16 @@ public function getAllSchemas()
...
+        return array_merge(parent::getColumnTypes(), [
+            'json',
+            'jsonb',
+            'xml',
+            'oid',
+            'cidr',
+            'inet',
+            'macaddr',
+            'interval',
+            ]);
```
Comment (dereuromark) [inline src/Phinx/Db/Adapter/PostgresAdapter.php]: indentation off

**Comment by dereuromark** [comment]
Any update on this?

**Comment by dereuromark** [comment]
ping @LevZaplatin

**Comment by GhaziTriki** [comment]
Would love to see this merged.

**Comment by dereuromark** [comment]
Can we resolve the conflicts or open a new PR? Then we can continue

**Comment by dereuromark** [comment]
@GhaziTriki Are you able to help out here?

**Pull request closed without merging**